cmd/soroban-cli/src/commands/contract/info.rs

```diff
@@ -49,26 +50,39 @@ pub enum Cmd {
     ///
     /// Outputs no data when no data is present in the contract.
     EnvMeta(env_meta::Cmd),
+
+    /// Output the contract build information, if available.
+    ///
+    /// If the contract has a meta entry like `source_repo=github:user/repo`, this command will try
+    /// to fetch the attestation information for the WASM file.
+    #[command(hide = true)]
```

Comment:
Instead of hiding this, I think it would also be fine to make it visible, and put the word "Experimental: " at the start of the comment above.

Reply:
And could elaborate in the longer comment text to say that the functionality, data verified, is likely to change and is still in development.

Reply:
@janewang wdyt?